test/resources/controls/aws_parallelcluster_install/install_packages_spec.rb

```diff
@@ -0,0 +1,52 @@
+# Use the name matching the resource type
+control 'install_packages' do
+  title 'Test installation of packages'
+
+  if virtualization.system == 'docker' && os.redhat? && os[:name] == 'redhat'
+    return
+  end
+
+  describe package('lvm2') do
```

Comment:
How did you decide what packages to include in your test?

Reply:
+1

Reply:
I search for a base packages shared between all the OSes